### PR TITLE
Integrate Ballot SC30 and SC31

### PR DESCRIPTION
**Pull request by sleevi**

### CODE REVIEW

**Comment by sleevi** [comment]
@dzacharo This follows the instructions provided at https://archive.cabforum.org/pipermail/infrastructure/2020-May/000223.html and mentioned in https://archive.cabforum.org/pipermail/infrastructure/2020-July/000276.html

I created a new branch in cabforum/documents, merged the SC30 and SC31 PRs into this branch, and then created a PR with both integrated. You can make whatever edits necessary, as Chair, to the relevant sections. It also ensures that the Travis CI will run and produce the versions of the documents (e.g. at https://cabforum-travis-artifacts.s3-us-west-2.amazonaws.com/builds/Ballot-SC30-and-31/BR.docx )

File: docs/EVG.md [hunk 1]
```diff
@@ -750,6 +761,18 @@ Before issuing an EV Certificate, the CA MUST ensure that all Subject organizati
...
 
+### 11.1.3.  Disclosure of Verification Sources
+
+Effective as of 1 October 2020, prior to the use of an Incorporating Agency or Registration Agency to fulfill these verification requirements, the CA MUST publicly disclose Agency Information about the Incorporating Agency or Registration Agency. This disclosure SHALL be through an appropriate and readily accessible online means.
+
+This Agency Information SHALL include at least the following:
+* Sufficient information to unambiguously identify the Incorporating Agency or Registration Agency (such as a name, jurisdiction, and website); and,
+* The accepted value or values for each of the `subject:jurisdictionLocalityName` (OID: 1.3.6.1.4.1.311.60.2.1.1), `subject:jurisdictionStateOrProvinceName` (OID: 1.3.6.1.4.1.311.60.2.1.2), and `subject:jursidictionCountryName` (OID: 1.3.6.1.4.1.311.60.2.1.3) fields, when a certificate is issued using information from that Incorporating Agency or Registration Agency, indicating the jurisdiction(s) that the Agency is appropriate for; and,
```
Comment (clintwilson) [inline docs/EVG.md:770]: I noticed that subject:jursidictionCountryName has a spelling error and should be subject:jurisdictionCountryName.

Comment (dzacharo) [reply docs/EVG.md:770]: @clintwilson, this should be added in the Cleanups and Clarifications ballot. I will update it.

**Comment by sleevi** [comment]
It’s already got a PR open to do this and will be part of it, now that I
confirmed with co-endorsers. You can close this.

**Review by dzacharo — APPROVED** [review]
Everything looks good.

**Comment by dzacharo** [comment]
@sleevi let me know if you want to take another look before this is merged to main.

**Pull request merged**